src/PKSim.Core/Services/GlobalPKAnalysisRunner.cs

```diff
@@ -24,13 +24,13 @@ public interface IGlobalPKAnalysisRunner
 
    public class GlobalPKAnalysisRunner : IGlobalPKAnalysisRunner
    {
-      private readonly ISimulationEngine<IndividualSimulation> _simulationEngine;
```

Comment:
This was I believe the wrong abstraction. (too deep). We have the SimulationRunner and it should be used by all other services